pkg/operator/controller/ingress/status.go

```diff
@@ -716,14 +715,8 @@ func checkDefaultCertificate(secret *corev1.Secret, domain string) error {
 		if err != nil {
 			continue
 		}
-		foundSAN := false
-		for i := range cert.DNSNames {
-			if cert.DNSNames[i] == domain {
-				foundSAN = true
-			}
-		}
-		if cert.Subject.CommonName == domain && !foundSAN {
```

Comment:
Removing this SAN upgrade logic seems problematic as we are backporting this in 4.15, so then 4.15 will not have this logic, but 4.16+ will have it. 

I think the logic needs to be removed in the master branch instead of removed in a backport like this. The 4.16 bug https://issues.redhat.com/browse/OCPBUGS-26498 is already ON_QA and we've branched to 4.17, so it's probably too late to merge this in master and try to backport it.

I think I should just handle this later when appropriate, keep this backport as "additive" only.

Reply:
Updated to not remove the SAN logic.